Assets/Scripts/Health/PlayerHealth.cs

```diff
@@ -18,10 +18,14 @@ public class PlayerHealth : MonoBehaviour
 
     private HealthManager healthManager;
 
+    bool isCheckingBurn;
```

Comment:
You never use it. I think it's fine to remove it cause similar check is made in HealthManager already, which is used in `healthManager.IsAllowedToBurn`

Reply:
I use it in IEnumerator

Reply:
oh wait im dumb, I change